doc/nrf/releases/release-notes-2.1.4.rst

```diff
@@ -0,0 +1,116 @@
+The changes affect Bluetooth® LE stack, Zephyr kernel, and :ref:`liblwm2m_carrier_readme` library.
+
+See :ref:`ncs_release_notes_214_changelog` for the complete list of changes.
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v2.1.4**.
+Check the :file:`west.yml` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`cloning_the_repositories` and :ref:`gs_updating_repos_examples` for more information.
+
+For information on the included repositories and revisions, see `Repositories and revisions for v2.1.4`_.
+
+IDE and tool support
+********************
+
+`nRF Connect for Visual Studio Code extension <nRF Connect for Visual Studio Code_>`_ is the only officially supported IDE for |NCS| v2.1.4.
+SEGGER Embedded Studio Nordic Edition is no longer tested or recommended for new projects.
+
+:ref:`gs_app_tcm`, used to install the |NCS| automatically from `nRF Connect for Desktop`_, is available for Windows, Linux, and macOS.
+
+Supported modem firmware
+************************
+
+See `Modem firmware compatibility matrix`_ for an overview of which modem firmware versions have been tested with this version of the |NCS|.
+
+Use the latest version of the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See :ref:`nrf9160_gs_updating_fw_modem` for instructions.
+
+Modem-related libraries and versions
+====================================
+
+.. list-table:: Modem-related libraries and versions
+   :widths: 15 10
+   :header-rows: 1
+
+   * - Library name
+     - Version information
+   * - Modem library
+     - `Changelog <Modem library changelog for v2.1.4_>`_
+   * - LwM2M carrier library
+     - `Changelog <LwM2M carrier library changelog for v2.1.4_>`_
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+See `known issues for nRF Connect SDK v2.1.4`_ for the list of issues valid for the latest release.
+
+.. _ncs_release_notes_214_changelog:
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+Protocols
+=========
+
+This section provides detailed lists of changes by :ref:`protocol <protocols>`.
+
+Bluetooth LE
+------------
+
+* Updated MPSL by removing calls to ``__WFE()`` for the nRF5340 SoC, to be in line with the Errata for the Anomaly 160.
+* Fixed a rare issue that could cause an assert when requesting timeslots with type ``MPSL_TIMESLOT_REQ_TYPE_EARLIEST``.
+
+For details, see the :ref:`MPSL changelog <nrfxlib:mpsl_changelog>`.
+
+Libraries
+=========
+
+This section provides detailed lists of changes by :ref:`library <libraries>`.
+
+Binary libraries
+----------------
+
+* :ref:`liblwm2m_carrier_readme` library:
+
+  * Added workaround for the known issue TNSW-46156.
```

Comment:
We can link to individual known issues, yes, with a ref tag -- see for example the individual link tag for DRGN-15852.

As the solution to my comment, I'd suggest the following edit to point the user to the list of known issues for the last version that included this known issue - so it is possible to read its description there.

```suggestion
  * Added workaround for the known issue TNSW-46156, last listed as active in the `known issues for nRF Connect SDK v2.1.3`_.
```

Reply:
@nordic-hani, FYI.

Reply:
Added this.